README.md

```diff
@@ -101,8 +94,8 @@ onCopy(text, result)
 
 
 #### `options`: PropTypes.shape({debug: bool, message: string})
-
-Optional [copy-to-clipboard](https://npm.im/copy-to-clipboard) options.
+ 
+Optional [copy-to-clipboard](https://npm.im/copy-to-clipboard) options. Here is a [demo](https://jinno.io/app/18/debug)
```

Comment:
How does jinno build happens? If it uses pre-compiled version then most likely it has outdated copy-co-clipboard lib baked in and has no `debug` option, if it uses proper impoers it would use the latest and would work. Currently I do not think `debug` option in jinno works.

Reply:
It's a complied version
This is the version that I have: ^5.0.3

Reply:
When I am looking at the Codepen demo it also doesn't work:
https://codepen.io/nkbt/pen/eNPoQv?editors=0010

I don't see the "Whoa" console

Reply:
Codepen definitely uses old version.

Reply:
I removed this param
I checked everything I can, it's just not working

I put this issue on resolve